src/Hedgehog/Property.fs

```diff
@@ -151,7 +151,7 @@ module private Pretty =
 
         List.iter (append sb) (Journal.toList journal)
 
-        sb.ToString(0, sb.Length - 1) // exclude extra newline
+        sb.ToString(0, sb.Length-1) // exclude extra newline
```

Comment:
```suggestion
        sb.ToString(0, sb.Length - 1)
```

Reply:
I think this shows up in the diff because of the removed space (but it shouldn't).

Reply:
Oh. I think we saw the sam thing at the same time. I already fixed it back.